content/blog/2020/06/2020-06-17-external-fingerprint-storage.adoc

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: "External Fingerprint Storage Phase-1 Updates"
+tags:
+- plugins
+- fingerprint
+- cloud-native
+- external-storage
+- developer
+- redis
+- gsoc
+author: stellargo
```

Comment:
would be great to add opengraph image and description

Reply:
My page profile photo and description is already set up, plus I ran it locally, it is rendering at the bottom of the page too

Reply:
Opengraph embeds image for social media and other channels (e.g. when you post a link in Slack), it is not about author.

See https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-blog-post for examples

Reply:
Oh, thank you! I have added the Jenkins gsoc image, is that okay?

Reply:
Yes, unless you want to add something more unique for better visibility. e.g. I used https://docs.google.com/presentation/d/1Q1PgNnRTgzBpVRXPqQo3PudzCa2eoc6_1_NRjFRMLrU/edit#slide=id.g80cdc72412_0_0 for the video recording

Reply:
That's so awesome, yes I will use that! In fact ill add it to the plugin repository too! Thanks for creating this :)